src/backend/drive/settings.py

```diff
@@ -419,6 +419,24 @@ class Base(Configuration):
     FRONTEND_FEEDBACK_ITEMS = values.DictValue(
         {}, environ_name="FRONTEND_FEEDBACK_ITEMS", environ_prefix=None
     )
+    FRONTEND_FEEDBACK_MESSAGES_WIDGET = values.BooleanValue(
```

Comment:
As it is a boolean, adding `ENABLED` somewhere in the name could be helpful I think

Reply:
And it could be added to the frontend configuration file made in PR https://github.com/suitenumerique/drive/pull/299 instead of having a growing list of settings related to the frontend application ?

Reply:
I'll put all the pre existing frontend stuff in this configuration file in a new PR + we need to setup a way to be able to use this file with the ANCT deployment, that we haven't yet (@sylvinus), that's why we need to still rely on envvar for now